library/bignum_core.h

```diff
@@ -74,6 +74,57 @@ size_t mbedtls_mpi_core_bitlen( const mbedtls_mpi_uint *A, size_t A_limbs );
 void mbedtls_mpi_core_bigendian_to_host( mbedtls_mpi_uint *A,
                                          size_t A_limbs );
 
+/**
+ * \brief   Perform a safe conditional copy of MPI which doesn't reveal whether
+ *          the condition was true or not.
+ *
+ * \param[out] X        The address of the first MPI. This must be initialized.
```

Comment:
For a copy, it's normal to have a `source` and `destination`, rather than `first` and `second` (using `first` and `second` doesn't make it clear which argument might be copied to which argument)